packages/main/src/themes/Avatar.css

```diff
@@ -210,4 +211,53 @@
 
 .ui5-avatar-initials {
 	color: inherit;
+}
+
+/* Slotted Badge */
+::slotted([slot="badge"]) {
+	position: absolute;
+	bottom: 0;
+	right: 0;
+	width: 1.125rem;
+	height: 1.125rem;
+	background: var(--sapButton_Emphasized_Background);
+	border: var(--sapButton_Emphasized_Background);
+	border-radius: 1rem;
+	color: var(--sapContent_BadgeTextColor);
+	justify-content: center;
+	font-size: var(--sapFontSmallSize);
```

Comment:
I think the badge should also have the font-family of 72 by default. [For example](https://openui5.hana.ondemand.com/test-resources/sap/ui/integration/demokit/cardExplorer/webapp/index.html#/explore/badge).